.github/workflows/ci_windows_mu4.yml

```diff
@@ -138,7 +142,7 @@ jobs:
       if: ${{ always() }}
       uses: actions/upload-artifact@v2
       with:
-        name: MU4_${{ github.run_id }}_Win${{ github.event.pull_request.title }}
+        name: ${{ env.UPLOAD_ARTIFACT_NAME }}
```

Comment:
Why not changing this for the 32bit and the portableApp builds too, even with those being commented out at the moment?
Maybe adding a Win32 / PortableApp prefix?

Reply:
I thought the commented out ones were not really up to date anyway, so initially didn't touch them.
But I updated them too, now.

Reply:
They will get used for Release and probably Beta, maybe Alpha too